### ISSUE TITLE
Problem with y axis when plotting data + best-model 

### ISSUE THREAD

**Issue by douglasalvesastro12** [bendichter/brokenaxes#74]
Hi, 

Can you help me out with this? Sorry I can't give details since it may be sensitive data (If you do need a piece of code I can edit it though)

When I plot just the data (in blue) I have x and y ticks ok, however, when I plot the model (dark line) on top of data apparently another set of x-y ticks are added. Is there a simple way to adjust these axis? I tried to turn off at least one of the plots axis? let's say the model x-y ticks with the command `bax.axes.yaxis.set_ticks([])
bax.axes.xaxis.set_ticks([])` but somehow it does not work as expected since the default axis shows up. I need to make the axis look good in the image attached. Also, the parallel bars representing the cutting is not exactly in the x-axis but slightly above it.

my code has these lines

fig = plt.figure(figsize=(10,7))
gs = gridspec.GridSpec(3, 2, width_ratios=[1.5,1])
...
bax = brokenaxes(xlims=(lc1_lims, lc2_lims), subplot_spec=gs[4])
data is plot with bax.errorbar() and model with bax.plot()
 
![Screenshot from 2021-08-29 21-48-04](https://user-images.githubusercontent.com/37444298/131274418-03f211d9-29e0-445e-b2f5-9d9c5f901eee.png)

**Comment by bendichter** [comment]
Could you reproduce the error using fake data? I don't quite have enough information here to help you

**Comment by douglasalvesastro12** [comment]
Hi @bendichter, I just figured it out. Just for the records, I created a figure with 6 subplots and called each ax1, ax2, so forth. Then set bax = brokenaxes(xlims=(lc1_lims, lc2_lims), subplot_spec=gs[4]) on one of these axis (aka gs[4]). The issue is that the ax4 I had previously created have default x-y ticks and on top of it bax make a plot. All I had to do was to suppress the ax4 x-y ticks ax4.set_axis_off(). A better way would be not creating these ax1, ax2, etc.. but I needed the broken axis feature only on one of my subplots. Something else I struggled with was the tight_layout command but managed to get around that. 

Did not know this package of yours, really cool. I'll be using it from now on. 

![Screenshot from 2021-08-30 17-49-55](https://user-images.githubusercontent.com/37444298/131410393-de73538e-3c41-460d-89ef-fccfc2536756.png)
